shared/src/metabase/shared/parameters/parameters.cljc

```diff
@@ -4,51 +4,65 @@
       (:clj
        [(:require [clojure.string :as str]
                   [metabase.mbql.normalize :as mbql.normalize]
-                  [metabase.shared.util.i18n :refer [trs]])]
+                  [metabase.shared.util.i18n :refer [trs]]
+                  [metabase.util.date-2 :as u.date]
+                  [metabase.util.date-2.parse.builder :as b]
+                  [metabase.util.i18n.impl :as i18n.impl])
+        (:import java.time.format.DateTimeFormatter)]
        :cljs
        [(:require ["moment" :as moment]
                   [clojure.string :as str]
                   [metabase.mbql.normalize :as mbql.normalize]
                   [metabase.shared.util.i18n :refer [trs]])]))
 
-(defn- formatted-list
-  [values]
-  (str (str/join ", " (butlast values)) " " (trs "and") " " (last values)))
+#?(:cljs (comment moment/keep-me))
```

Comment:
Our namespace check linter was incorrectly detecting `moment` as being unused so I've added this line to fool it. (We have a similar thing in `shared/src/metabase/shared/util/i18n.cljs`)

The linter (or more accurately, the `refactor.nrepl` middleware) wasn't complaining before because the clojure and clojurescript ns forms were otherwise identical. Once I added more imports to the clojure form the error was surfaced. I'm still not 100% sure why that matters but I'm pretty sure there's a bug in that middleware somewhere.

Reply:
A comment to this effect should probably go in the code, rather than the review?